test/test_ops.py

```diff
@@ -1000,5 +1006,34 @@ def gen_iou_check(box, expected, tolerance=1e-4):
             gen_iou_check(box_tensor, expected, tolerance=0.002 if dtype == torch.float16 else 1e-3)
 
 
+class TestStochasticDepth:
+    @pytest.mark.skipif(stats is None, reason="scipy.stats not available")
+    @pytest.mark.parametrize('mode', ["batch", "row"])
+    @pytest.mark.parametrize('p', [0.2, 0.5, 0.8])
+    def test_stochastic_depth(self, mode, p):
+        random.seed(42)
```

Comment:
Shouldn't this be the torch RNG that we want to fix? I see calls to `torch.rand` in `stochastic_depth()`

Reply:
That's what I thought as well but then I saw the refactored version of the transforms and followed the same approach:
https://github.com/pytorch/vision/blob/3fa2055b5a4807a8ffc577c53da180382cafd17c/test/test_transforms.py#L522-L524

Do you think this was a mistake introduced during the refactoring?

Reply:
It looks like these lines were introduced in https://github.com/pytorch/vision/pull/3123

Perhaps I'm missing what this is supposed to do then: are we trying to fix the randomness of `StochasticDepth`, or the randomness of something else, and what is this "something else"?

As far as I understand the test procedure in itself (the call to scipy.stats) is deterministic?

Reply:
> It looks like these lines were introduced in #3123

This is before #3123. See [here](https://github.com/pytorch/vision/blame/4eab7a67f9f134c810eb7d1977e3ef9aa3fa1b26/test/test_transforms.py#L435-L445). Perhaps the intention was not to fix at all the rand on torch and only work on PIL/numpy randomness. I'll check if I can completely remove it.

> As far as I understand the test procedure in itself (the call to scipy.stats) is deterministic?
Yes, that's correct.

Reply:
It seems it passes without fixing the seed, so I removed it. If we see flakiness we can set the seed for both all randgens.